### DISCUSSION TITLE
Issue with DST for the lock schedules

### DISCUSSION THREAD

**Discussion by mohanblaze** [zwave-js/zwave-js#6700] [Feature Requests]
### Description

The current scheduling feature for the lock is functioning effectively, and we are pleased with its performance. 

However, we have encountered an issue related to Daylight Saving Time (DST). We configured yearly and daily schedules in February, using the timezone America/Montevideo. 

These schedules were successfully tested and worked until March 10th, when DST began, causing them to stop working at the expected times. What is the solution for this? Can you please provide us with a solution for this?

**Comment by AlCalzone** [comment]
I don't think there's a general solution. The scheduling is handled by the lock itself, all Z-Wave JS does is set up the schedules using existing commands. So if the schedules don't respect DST, you'll have to ask the lock manufacturer why that's the case.
What I can imagine:
- The lock does not understand/support DST or its implementation is flawed
- Timezone information hasn't been set up on the lock
- If the lock regularly requests the time from the hub, the hub may be set to the wrong timezone
